pvr.plutotv/resources/language/resource.language.de_de/strings.po

```diff
@@ -30,6 +30,10 @@ msgctxt "#30007"
 msgid "(Re)install Widevine CDM library"
 msgstr "Widevine CDM-Bibliothek (erneut) installieren"
 
+msgctxt "#30008"
+msgid "First channel number"
+msgstr "Erste Sendernummer"
+
```

Comment:
Please remove this from this PR, as translations are handled via Weblate, not GitHub. Only "source" language, which is en_gb is handled via GitHub.